### PR TITLE
Werkzeug mentions

### LINKED ISSUE #190: Should link to Werkzeug’s docs prominently
Many Flask questions on the mailing list or IRC are easily answered by reading Werkzeug documentation. Maybe Flask docs should make more obvious (and repeat everywhere) that many thing (Request and Response objects, routing, ...) are just inherited from Werkzeug, and link to the relevant documentation.

### LINKED ISSUE #1625: Documentation / Quickstart / Debugger - how to activate
When following the quickstart, specifically:

https://github.com/mitsuhiko/flask/search?utf8=%E2%9C%93&q=++Screenshot+of+the+debugger+in+action

I found myself jumping to google to learn how to activate the debugger. Perhaps if the screen grab showed & highlighted the console icon on the far left of my screen, that wouldn't have happened.

I'm happy to submit a pull request with a replacement screenshot if you agree this would be useful, noting I'm not on mac, so screen grab might look out of place, if that detail is considered important

### PR DESCRIPTION
**Pull request by doobeh**
Debugger docs, response objects and url converters. Aims to fix #1625 and fix #190.

---
This change is [Reviewable](https://reviewable.io/reviews/pallets/flask/1857)

### CODE REVIEW

File: flask/wrappers.py [hunk 1]
```diff
@@ -194,11 +194,13 @@ def _load_form_data(self):
 
 class Response(ResponseBase):
     """The response object that is used by default in Flask.  Works like the
-    response object from Werkzeug but is set to have an HTML mimetype by
-    default.  Quite often you don't have to create this object yourself because
+    :class:`~werkzeug.wrappers.Response` object from Werkzeug but is set to
+    have an HTML mimetype by default.  Quite often you don't have to create
```
Comment (jeffwidman) [inline flask/wrappers.py]: Should 'mimetype' be capitalized? I really don't know...

File: docs/quickstart.rst [hunk 2]
```diff
@@ -197,18 +202,39 @@ function.  Optionally a converter can be used by specifying a rule with
         # show the post with the given id, the id is an integer
         return 'Post %d' % post_id
 
-The following converters exist:
+Flask builds on the routing system provided by Werkzeug which includes
+the following build other converter types.
```
Comment (jeffwidman) [inline docs/quickstart.rst]: wording is a little kludgey

File: docs/quickstart.rst [hunk 3]
```diff
@@ -197,18 +202,39 @@ function.  Optionally a converter can be used by specifying a rule with
...
+`path` (:class:`~werkzeug.routing.PathConverter`)         like the default but also accepts slashes
+`any` (:class:`~werkzeug.routing.AnyConverter`)           matches one of the items provided
+`uuid` (:class:`~werkzeug.routing.UUIDConverter`)         accepts UUID strings
+========================================================  ===============================================
+
+Building custom converters is also possible by subclassing
+the :class:`~werkzeug.routing.BaseConverter` then registering your
+custom converter against :meth:`flask.Flask.url_map.converters`::
+
+    from werkzeug.routing import BaseConverter
```
Comment (untitaker) [inline docs/quickstart.rst]: I think this is out of scope for the quickstart

Comment (jeffwidman) [inline docs/quickstart.rst]: Agreed

**Comment by untitaker** [comment]
@doobeh Could you take a look at the inline comments? Thanks!

File: docs/quickstart.rst (hunk 3 above)
Comment (davidism) [inline docs/quickstart.rst]: I merged a PR a while ago that expanded this list in the first place, should have caugth that.  It needs to be *reduced* to an actual quickstart here and link to the api docs:

> The default converter is `string`.  Other converters are available as well, see [URL route registrations](http://flask.pocoo.org/docs/0.11/api/#url-route-registrations) to learn more.  Here are some examples:

The api docs should include all this new information.

File: flask/wrappers.py (hunk 1 above)
Comment (davidism) [inline flask/wrappers.py]: I think it's fine, it just mimics the attribute name.

File: docs/quickstart.rst (hunk 3 above)
Comment (untitaker) [inline docs/quickstart.rst]: I disagree, I think only writing own converters is out of scope for the
quickstart.

Comment (davidism) [inline docs/quickstart.rst]: OK, that's fine.  The quickstart should still link to the api docs, and the api docs need the converter links and custom converter information.

**Comment by untitaker** [comment]
Any updates on this?

**Comment by untitaker** [comment]
I'm closing this due to inactivity, feel free to refile!
